### PR TITLE
feat: [WIP] added home page with basic notes listing features

### PR DESCRIPTION
**Pull request by kedark152**
### feat: [WIP] added home page with basic notes listing features

**Preview Link: https://deploy-preview-2--notes-master.netlify.app/home**

**Features added in this Pull Request:-**
In Home Page Below are the Listed features:-
- Create a new Note.
- Edit the Note.
- Delete the Note.
- Pin/Unpin the Note while editing & on listing page.
- added date of note creation.

Updated styling for landing page.

### CODE REVIEW

File: src/components/EditNotesCard.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  console.log(notesState);
+
+  const [errorText,setErrorText] = useState("");
+
+  const handleSaveForm = ()=>{
+    const form = editNotesForm.current;
+      if(form["editTitle"].value.length>0 && form["editBody"].value.length>0){
+          setErrorText("");
+          return {_id: uuid(),title:form["editTitle"].value,body:form["editBody"].value,editBoxStatus:"hide-edit-box",isPinned:notesState.isPinned};
+    }
```
Comment (Hamdrive) [inline src/components/EditNotesCard.js]: Do implement the use of code formatter, will make reviewing and code readability much easier

File: src/styles/component/editnotes.css [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    color: var(--black);
+    background-color: var(--grey-lightest);
+}
+.top-section i{
+  margin-left: auto;
+}
+
+.edit-title{
+    font-size: 1.6rem;
+    padding: 0.2rem;
```
Comment (Hamdrive) [inline src/styles/component/editnotes.css]: small units do use px instead of rem

File: src/styles/component/editnotes.css [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+.show-edit-box{
+    display: grid;
+}
+.hide-edit-box{
+    display: none;
+}
+.error-text{
+    color: red
+}
+textarea, textarea:focus
```
Comment (Hamdrive) [inline src/styles/component/editnotes.css]: ideally do not directly mutatethe HTML tag, instead apply the styles by classname on textarea

File: src/styles/layouts/navbar.css [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+    top: 20%;
+    right: 2%; 
+}
+
+#search-bar{
+    border: none;
+    height: 2.5rem;
+    width: 75%;
+    font-size: 1.2rem;
+    padding: 0rem 0.5rem;
```
Comment (Hamdrive) [inline src/styles/layouts/navbar.css]: dont have to explicitly mention rem for zero value

File: src/styles/pages/home.css [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+.cell-block a:visited {
+   text-decoration: none; 
+  color: var(--indigo-color);
+}
+
+/* #cell:visited, #cell:hover, #cell:focus {
+    text-decoration: none;
+    color: black;
+} */
+
```
Comment (Hamdrive) [inline src/styles/pages/home.css]: Cleanup commented code

**Review by Hamdrive — COMMENTED** [review]
Great work implementing basic logic for note taking, styles look good too
Will suggest to always format code to ensure better readability and also eases the PR review process
Have suggested some edits do look into them

File: src/styles/pages/home.css (hunk 5 above)
Comment (kedark152) [reply src/styles/pages/home.css]: Done changes in new commit

File: src/styles/layouts/navbar.css (hunk 4 above)
Comment (kedark152) [reply src/styles/layouts/navbar.css]: Done changes in new commit

File: src/styles/component/editnotes.css (hunk 3 above)
Comment (kedark152) [reply src/styles/component/editnotes.css]: Done changes in new commit

File: src/styles/component/editnotes.css (hunk 2 above)
Comment (kedark152) [reply src/styles/component/editnotes.css]: Done changes in new commit

File: src/components/EditNotesCard.js (hunk 1 above)
Comment (kedark152) [reply src/components/EditNotesCard.js]: Thanks for your Review. will implement changes in new commit.

File: src/components/EditNotesCard.js [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+import {useRef, useState} from "react";
+import "../styles/component/editnotes.css"
+import "../styles/utils/variable.css"
+import { useNotes } from "../context/notes-context"
+import { v4 as uuid } from "uuid"
+
+export const EditNotesCard = () =>{
+  const editNotesForm = useRef(null);
+  const {notesState,dispatchNotes} = useNotes();
+  console.log(notesState.setEditBox);
```
Comment (rushikesh3299) [inline src/components/EditNotesCard.js]: remove console logs while pushing code

File: src/components/Navbar.js [hunk 7]
```diff
@@ -0,0 +1,30 @@
+import "../styles/layouts/navbar.css"
+import { Link } from "react-router-dom"
+
+export const Navbar = () =>{
+    return (<>
```
Comment (rushikesh3299) [inline src/components/Navbar.js]: Do use code formatters with vscode

**Review by rushikesh3299 — APPROVED** [review]
Overall looks good to me

File: src/components/EditNotesCard.js (hunk 6 above)
Comment (kedark152) [reply src/components/EditNotesCard.js]: Thanks, Done Changes in new commit.

File: src/components/Navbar.js (hunk 7 above)
Comment (kedark152) [reply src/components/Navbar.js]: Thanks, Done Changes in new commit.

**Comment by kedark152** [comment]
Thanks Hamza & Rushikesh for doing my PR Review. made some changes suggested and merging the PR.

**Pull request merged**